addons/web/static/tests/_framework/mock_server/mock_server.js

```diff
@@ -109,6 +109,7 @@ const authenticateUser = (user) => {
     }
     env.cookie.set("sid", user.id);
     env.uid = user.id;
+    user.userId = env.uid;
```

Comment:
i don't think we want this

cc @Arcasias

Reply:
I removed it , I didn't know that env.uid updates the user.userId too